src/index.js

```
@@ -389,7 +389,33 @@ const wasDeltaMadeByAuthor = comment => comment.link_author === getCommentAuthor
 /* Invoked after the DeltaLog post is made, so `deltaLogKnownPosts` will be populated */
 const deltaLogStickyTemplate = _.template(i18n[locale].deltaLogSticky)
 const findOrMakeStickiedComment = async (linkID, comment, deltaLogPost) => {
-  if (deltaLogPost.stickiedCommentID) {
+  const stickyID = deltaLogPost.wikientry.stickiedCommentID
+  const opName = deltaLogPost.postentry.opUsername
+  const deltsAwardedByOP = deltaLogPost.postentry.comments.filter(
+    comm => comm.awardingUsername === opName
+  ).length
+
+  if (stickyID) {
+    // Update the N in 'OP has awarded N deltas...'
+    const stickyCommentBody = deltaLogStickyTemplate({
+      username: opName,
+      linkToPost: `/r/${deltaLogSubreddit}/comments/${deltaLogPost.wikientry.deltaLogPostID}`,
+      deltaLogSubreddit,
+      opawarded: deltsAwardedByOP,
+    })
+
+    const updateParams = {
+      text: `${stickyCommentBody}`,
+      thing_id: `${stickyID}`,
+    }
+
+    const updateResponse = await reddit.query({
+      URL: `/api/editusertext?${stringify({ thing_id: `${stickyID}` })}`,
+      method: 'POST',
+      body: stringify(updateParams),
+    })
+    if (updateResponse.error) throw Error(updateResponse.error)
```

I don't think you would want to throw this error. Logging it would probably be enough.